src/main/java/com/buglife/sdk/IOUtils.java

```
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2017 Buglife, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.buglife.sdk;
+
+import android.support.annotation.Nullable;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.Closeable;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class IOUtils {
+    private IOUtils() {/* No instances */}
+
+    static void writeStringToFile(String data, File file) {
+        OutputStream output = null;
+        InputStream input = null;
+        try {
+            input = new ByteArrayInputStream(data.getBytes());
+            output = new FileOutputStream(file);
+            write(input, output);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Swallowed exceptions 😬